plugin-tester-scala/src/main/scala/example/myapp/helloworld/MtlsGreeterServer.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2018-2023 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+//#full-server
+package example.myapp.helloworld
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.ConnectionContext
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.HttpsConnectionContext
+import akka.http.scaladsl.model.AttributeKeys
+import akka.http.scaladsl.model.HttpRequest
+import akka.http.scaladsl.model.HttpResponse
+import akka.pki.pem.DERPrivateKeyLoader
+import akka.pki.pem.PEMDecoder
+import com.typesafe.config.ConfigFactory
+import example.myapp.helloworld.grpc._
+import org.slf4j.LoggerFactory
+
+import java.security.KeyStore
+import java.security.SecureRandom
+import java.security.cert.Certificate
+import java.security.cert.CertificateFactory
+import javax.net.ssl.KeyManagerFactory
+import javax.net.ssl.SSLContext
+import javax.net.ssl.TrustManagerFactory
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.io.Source
+
+object MtlsGreeterServer {
+  def main(args: Array[String]): Unit = {
+    // Important: enable HTTP/2 in ActorSystem's config
+    // We do it here programmatically, but you can also set it in the application.conf
+    val conf = ConfigFactory
+      .parseString("akka.http.server.preview.enable-http2 = on")
```

Comment:
we changed the `preview`, or is this with older Akka HTTP?

Reply:
Correct, but it seems the full bump was missed in akka-grpc, because we have Akka HTTP 10.5.0-M1 as a dependency also for relased versions, dang.

Reply:
Fixing separately https://github.com/akka/akka-grpc/pull/1782